vagrantcdk/lab1/chapter1.md

```diff
@@ -38,6 +38,8 @@ Perform the following commands as root unless instructed otherwise.
 
 ```bash
 cat /usr/lib/systemd/system/docker.service
+cat /usr/lib/systemd/system/docker-storage-setup.service
+cat /usr/lib/systemd/system/docker-registry.service
```

Comment:
hey @scollier - I think I'm going to remove the docker-registry.service from this.. I don't think it's relevant for our students and we cover the registry later.